### PR TITLE
add allure.testcase decorator

### PR DESCRIPTION
**Pull request by Headless1**

### CODE REVIEW

File: tests/guest/test_home_page.py [hunk 1]
```diff
@@ -12,6 +14,10 @@
 from SpaceToStudy.ui.pages.home_page.home_guest import HomePageGuest, BUTTON_GET_STARTED_FOR_FREE
 from tests.test_runners import BaseTestRunner
 
+TEST_CASE_176 = "https://github.com/UA-1023-TAQC/SpaceToStudyTA/issues/176"
+TEST_CASE_175 = "https://github.com/UA-1023-TAQC/SpaceToStudyTA/issues/175"
+TEST_CASE_198 = "https://github.com/UA-1023-TAQC/SpaceToStudyTA/issues/198"
```
Comment (lhalam) [inline tests/guest/test_home_page.py]: remove this

File: tests/guest/test_home_page.py [hunk 2]
```diff
@@ -193,6 +199,7 @@ def test_the_collapse_block_ui_text(self):
                           .get_description())
         self.assertIn(description_dc, description)
 
+    @allure.testcase(TEST_CASE_176)
```
Comment (lhalam) [inline tests/guest/test_home_page.py]: set url

File: tests/guest/test_home_page.py [hunk 3]
```diff
@@ -206,6 +213,7 @@ def test_what_can_you_do_elements_visible(self):
                                .get_btn())
         self.assertIsNotNone(become_tutor_button, "The 'Become a tutor' button is not found")
 
+    @allure.testcase(TEST_CASE_175)
```
Comment (lhalam) [inline tests/guest/test_home_page.py]: set url

File: tests/guest/test_home_page.py [hunk 4]
```diff
@@ -241,6 +249,7 @@ def test_open_who_we_are_block_by_tabs(self):
         title = (HomePageGuest(self.driver).get_who_we_are_block().get_title())
         self.assertEqual("Who we are", title)
 
+    @allure.testcase(TEST_CASE_198)
```
Comment (lhalam) [inline tests/guest/test_home_page.py]: set url

File: tests/guest/test_home_page.py [hunk 5]
```diff
@@ -253,8 +262,10 @@ def test_that_controls_active_after_navigating_to_them(self):
         ActionChains(self.driver).move_to_element(become_a_student).perform()
         sleep(5)
         button_after_it_is_hovered_over = become_a_student.value_of_css_property("background-color")
-        self.assertNotEqual(button_before_it_is_hovered_over, button_after_it_is_hovered_over, "The button hasn't changed")
+        self.assertNotEqual(button_before_it_is_hovered_over, button_after_it_is_hovered_over,
+                            "The button hasn't changed")
 
+    @allure.testcase(TEST_CASE_198)
```
Comment (lhalam) [inline tests/guest/test_home_page.py]: set url

**Review by lhalam — COMMENTED** [review]
update

File: tests/guest/test_home_page.py (hunk 1 above)
Comment (Headless1) [reply tests/guest/test_home_page.py]: done

File: tests/guest/test_home_page.py (hunk 2 above)
Comment (Headless1) [reply tests/guest/test_home_page.py]: done

File: tests/guest/test_home_page.py (hunk 3 above)
Comment (Headless1) [reply tests/guest/test_home_page.py]: done

File: tests/guest/test_home_page.py (hunk 4 above)
Comment (Headless1) [reply tests/guest/test_home_page.py]: done

File: tests/guest/test_home_page.py (hunk 5 above)
Comment (Headless1) [reply tests/guest/test_home_page.py]: done

**Pull request merged**